packages/xo-server-sdn-controller/src/index.js

```diff
@@ -454,6 +588,44 @@ class SDNController extends EventEmitter {
         })
         poolNetwork.starCenter = pif.host
         this._starCenters.set(host.$id, host.$ref)
+
+        const crossPoolNetwork = find(
+          this._crossPoolNetworks,
+          crossPoolNetwork =>
+            crossPoolNetwork.networks.includes(pif.$network.$ref)
+        )
+        if (crossPoolNetwork !== undefined) {
+          if (crossPoolNetwork.poolCenter !== undefined) {
+            const centerPoolNetwork = this._getPoolNetwork(
+              crossPoolNetwork.poolCenter,
+              crossPoolNetwork
+            )
+            await this._resetCrossPoolConnections(
+              poolNetwork,
+              crossPoolNetwork.uuid
```

Comment:
Isn't it what `crossPoolNetwork.poolCenter` already is?

Reply:
I don't understand your question.

Reply:
`crossPoolNetwork.poolCenter` is a ref, not an object.